docs/run/run-primitives-batch.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Run primitives in a batch
```

Comment:
Why is this titled Run _primitives_ in a batch, whereas the other is Run _jobs_ in a session ?

Reply:
Good question - seems like they should be consistent.  @javabster do you have a preference?